sentry_sdk/integrations/flask.py

```diff
@@ -87,10 +96,14 @@ def sentry_patched_wsgi_app(self, environ, start_response):
             if Hub.current.get_integration(FlaskIntegration) is None:
                 return old_app(self, environ, start_response)
 
-            return SentryWsgiMiddleware(lambda *a, **kw: old_app(self, *a, **kw))(
+            patched_app = SentryWsgiMiddleware(lambda *a, **kw: old_app(self, *a, **kw))(
                 environ, start_response
             )
 
+            patched_app.jinja_env.globals['sentry_trace'] = self._get_sentry_trace
```

Comment:
there's a flask signal like `got_first_request` or something like that which would probably be better for patching in the jinja globals. as-is, this config change is done on every request

Reply:
Ugh, good point. Let me check.

Reply:
So, there is `before_first_request` that said we'll be calling that on the new app instance anyway?

Reply:
Okay, this should be fixed now so resolving.